packages/protocol/script/DeployOnL1.s.sol

```diff
@@ -383,6 +386,18 @@ contract DeployOnL1 is DeployCapability {
                 ProverSet.init, (owner, vm.envAddress("PROVER_SET_ADMIN"), rollupAddressManager)
             )
         });
+
+        RiscZeroGroth16Verifier risc0Verifier =
+            new RiscZeroGroth16Verifier(ControlID.CONTROL_ROOT, ControlID.BN254_CONTROL_ID);
+
+        deployProxy({
+            name: "tier_sgx_zkvm",
```

Comment:
```suggestion
            name: "risk_zero_verifier",
```

Reply:
zk instead of sgx+ zk for the time being, as the verifier for sgx+zk is not yet available

Reply:
The contract for the verifier of sgx+zk and the corresponding client-side changes will be added in the next iteration. The changes of contract need to be made by @adaki2004.

Reply:
> The contract for the verifier of sgx+zk and the corresponding client-side changes will be added in the next iteration. The changes of contract need to be made by @adaki2004.

Not sure i know or understand what you refer and expect from me but sure i can help, but would be cool if we can sync offline/discord

Reply:
> Not sure i know or understand what you refer and expect from me but sure i can help, but would be cool if we can sync offline/discord

basically it's the same requirement I mentioned to you days ago, for we need a zk+sgx verifier which receives both sgx & zk proof and give positive result if both pass.

Reply:
> > Not sure i know or understand what you refer and expect from me but sure i can help, but would be cool if we can sync offline/discord
> 
> basically it's the same requirement I mentioned to you days ago, for we need a zk+sgx verifier which receives both sgx & zk proof and give positive result if both pass.

Ah ok. We already had a similar of it. 

@YoGhurt111  On branch main, go back in history.
e.g.:
`git checkout 78f04812de1bcb22ed40c9ae9b16e42d3d3783c2`

You will see a TEE vs ZK (sgx + risc zero) contract dispatcher. But in case you need help would be cool to get more info on how to dispatch exactly (between sgx, tdx, or risc0, sp1)

Reply:
OK, I will try that with the corresponding client-side changes in the next iteration.

Reply:
This shall simply be called "risc0_verifier", not "sgx_risc0_verifier"

Reply:
Fixed it.